src/libs/actions/Session/index.js

```diff
@@ -90,11 +90,33 @@ function signOutAndRedirectToSignIn() {
  * @param {String} [login]
  */
 function resendValidationLink(login = credentials.login) {
-    Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: true});
-    DeprecatedAPI.ResendValidateCode({email: login})
-        .finally(() => {
-            Onyx.merge(ONYXKEYS.ACCOUNT, {isLoading: false});
-        });
+    const optimisticData = [{
+        onyxMethod: CONST.ONYX.METHOD.MERGE,
+        key: ONYXKEYS.ACCOUNT,
+        value: {
+            isLoading: true,
+            errors: null,
+            message: '',
```

Comment:
Shouldn't this be `null`?

Reply:
why `null` instead of `''`? 

Looking back at the doc we didn't have `message` or `errors` in optimistic data so I might have just overlooked something when writing it https://docs.google.com/document/d/1i8YQApRXOC1vWl4LMBUq4vHmA-58rNtVwn8gWpvLao0/edit#bookmark=id.eb2jmf7ipcs1

Reply:
> we're using Pattern C here so we don't really need optimistic data to have a message, right

@bondydaa 

riiight, setting message to empty string is more to clear it out if the user re-clicks the button to send themselves another validation link. @aldo-expensify requested the change [here](https://github.com/Expensify/App/pull/10556#discussion_r954767368), and I personally think it makes sense.

As for `null` vs `''` it doesn't really matter what we set it to, since both are falsey and will prevent [this](https://github.com/Expensify/App/blob/e9920b7f2d494bfbdf444480986e97458027fb33/src/pages/signin/ResendValidationForm.js#L74-L76) from displaying. I've updated it to `null` to keep it more in line with other actions that perform similar stuff in `optimisticData`.